demos/asciidoc/didact-demo.didact.adoc

```diff
@@ -1,3 +1,6 @@
+//attributes
```

Comment:
what's the meaning of this `//attributes` comment?

Reply:
it is a common section used to define a collection of attributes at the beginning of an adoc file. a writing convention